bom/application/pom.xml

```
@@ -138,7 +138,7 @@
         <jboss-logging.version>3.4.1.Final</jboss-logging.version>
         <mutiny.version>0.13.0</mutiny.version>
         <mutiny-vertx.version>1.4.0</mutiny-vertx.version>
-        <kafka2.version>2.5.0</kafka2.version>
+        <kafka2.version>2.6.0</kafka2.version>
```

```suggestion
        <kafka2.version>2.6.1</kafka2.version>
```

2.6.1 is out by now.

---

Ah ah, I just updated it.